### PR TITLE
[argp_standalone] Add argp_standalone for musl

### PR DESCRIPTION
**Pull request by jpsamaroo**

### CODE REVIEW

File: A/argp_standalone/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                  "dec79694da1319acd2238ce95df57f3680fea2482096e483323fddf3d818d8be"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/argp-*/
+CFLAGS="-fPIC" ./configure --prefix=${prefix} --host=${target}
+make -j${nproc}
+install -D -m644 argp.h $WORKSPACE/destdir/include/argp.h
+install -D -m755 libargp.a $WORKSPACE/destdir/lib/libargp.a
```
Comment (giordano) [inline A/argp_standalone/build_tarballs.jl:20]:
```suggestion
install -D -m644 argp.h ${includedir}/argp.h
install -D -m755 libargp.a ${libdir}/libargp.a
```

File: A/argp_standalone/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+platforms = [p for p in supported_platforms() if Sys.islinux(p) && libc(p) == "musl"]
+
+# The products that we will ensure are always built
+products = [
+    FileProduct("lib/libargp.a", :libargp),
+    FileProduct("include/argp.h", :argp_h),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
```
Comment (giordano) [inline A/argp_standalone/build_tarballs.jl:33]:
```suggestion
dependencies = Dependency[
```

File: A/argp_standalone/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "argp_standalone"
+version = v"1.3.4"
```
Comment (giordano) [inline A/argp_standalone/build_tarballs.jl:6]: Shouldn't this be
```suggestion
version = v"1.3.0"
```
?

**Review by giordano — COMMENTED** [review]
The package doesn't have a license?

**Comment by jpsamaroo** [comment]
From the APKBUILD, it's GPL2. How do I specify that manually?

**Comment by giordano** [comment]
It's annoying there is no license anywhere... You need to do something like this: https://github.com/JuliaPackaging/Yggdrasil/blob/e52b984ec81a366ff2a5ed015dac80d368ad47ab/T/Tokei/build_tarballs.jl#L12
get the license file from somewhere, add it as a source, install it with [`install_license`](https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file)

File: A/argp_standalone/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+using BinaryBuilder
+
+name = "argp_standalone"
+version = v"1.3.0"
+
+# Collection of sources required to build argp-standalone
+sources = [
+    ArchiveSource("http://www.lysator.liu.se/~nisse/misc/argp-standalone-$(version.major).$(version.minor).tar.gz",
+                  "dec79694da1319acd2238ce95df57f3680fea2482096e483323fddf3d818d8be"),
+    FileSource("https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt", "8177f97513213526df2cf6184d8ff986c675afb514d4e68a404010521b880643"; filename="LICENCE"),
```
Comment (giordano) [inline A/argp_standalone/build_tarballs.jl:12]: :gb: 
```suggestion
    FileSource("https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt", "8177f97513213526df2cf6184d8ff986c675afb514d4e68a404010521b880643"; filename="LICENSE"),
```

Comment (jpsamaroo) [reply A/argp_standalone/build_tarballs.jl]: :smile:

Comment (jpsamaroo) [reply A/argp_standalone/build_tarballs.jl]: Sorry, accidentally force-pushed over your changes...

**Pull request merged**